controllers/suggestionsController.js

```diff
@@ -16,29 +16,28 @@ class SuggestionsController {
 
 	/*
 	 This function chains github API calls in order to create a pull request
-	 @param accessToken: A valid accessToken to access the github API
-	 @param title: (string) Will be used as a title for the pull request creation
-	 @param description: (urlencoded JSON string) The state we want to record
-	 @param state: (urlencoded JSON string) The state we want to record
+	 @param holder.title: (string) Will be used as a title for the pull request creation
+	 @param holder.description: The full commit message and PR initial comment
+	 @param holder.state: (urlencoded JSON string) The state we want to record
 	 @param res: An expressjs Response object to get back to the user
 	 */
-	createPullRequest(title, description, state, res) {
+	createPullRequest(holder, res) {
```

Comment:
This is not an acceptable name: anything can be a “holder”. You might as well call this “obj” or “lkjsdklsjglqskjg“ :wink:

What does this object model? What concept has a title, a description and a state? Is it a “testCase”? Is it a “testRun”? A “testSuggestion” maybe? 🙂